common/config/rush/pnpm-lock.yaml

```diff
@@ -2405,7 +2403,7 @@ packages:
     resolution: {integrity: sha512-ALYone6pm6QmwZoAgeyNksccT9Q4AWZQ6PvfwR37GT6r6FWUPguq6sUmNGSMV2Wr761oQoBxwGGa6DR5o1DC9g==}
     dependencies:
       '@types/connect': 3.4.35
-      '@types/node': 12.20.50
+      '@types/node': 17.0.31
```

Comment:
unfortunate version dance that we couldn't figure out why it happens and how to avoid it :(

Reply:
`rush update --recheck` doesn't help? :(

Reply:
no, rush update -full doesn't touch these, but brings in more other upgrades